lib/API/MTL/MTLDevice.cpp

```diff
@@ -58,9 +58,9 @@ class MTLDevice : public offloadtest::Device {
   struct InvocationState {
     InvocationState() { Pool = NS::AutoreleasePool::alloc()->init(); }
     ~InvocationState() {
-      for (auto T : Textures)
+      for (auto *T : Textures)
         T->release();
-      for (auto B : Buffers)
+      for (auto *B : Buffers)
```

Comment:
Better to spell out `MTL::Texture *` and `MTL::Buffer *` here rather than use auto